pyproject.toml

```diff
@@ -29,6 +29,9 @@ dependencies = [
     'typing_extensions; python_version < "3.7"',
     'pyyaml',
     'tomli >= 1.1.0; python_version < "3.11"',
+    'httpx',
+    'distro',
+    'pyhumps'
```

Comment:
These should be claimed in bohrium-sdk. I don't think they should be claimed in dpdispatcher